build-scripts/manylinux-container-image/Dockerfile

```diff
@@ -5,8 +5,8 @@ ARG LIBSSH_VERSION=0.9.6
 MAINTAINER Python Cryptographic Authority
 WORKDIR /root
 
-ADD update-packages.sh /root/update-packages.sh
-RUN ./update-packages.sh
+# ADD update-packages.sh /root/update-packages.sh
```

Comment:
why?

Reply:
This was me trying to work around the _2_28 images being AlmaLinux which doesn't have `prelink` available to install, and also the debian repos for the older images having been moved to archive (and thus 404ing without more changes).

I wanted to see if we could get away without calling out to the package manager at all, and it looks like we can, so the next step to me seems to be removing the script entirely. I just didn't want to go through all of that and then have to fish it back out of the history if the images depended further on the package manager.

Reply:
Historically, the manylinux image generation has been `git merge`d from the upstream, which is why I was aiming at keeping the differences minimal, so there are less conflicts on sync.